keyboards/45rstandard/kb.h

```diff
@@ -0,0 +1,18 @@
+#ifndef KB_H
```

Comment:
Please also rename this file to `45rstandard.h`.

Reply:
Thank you very much.